docs/terraformoptions.adoc

```diff
@@ -407,6 +407,16 @@ EOT
 | RUNNING/STOPPED
 | RUNNING
 
+|enable_operator_pv_encryption_in_transit
+|Whether to encrypt data in transit between the instance, the boot volume, and the block volumes in Operator.
+|true/false
+|false
+
+|operator_volume_kms_id
```

Comment:
If there's a dependency on a variable defined to be of a certain value, please mention it here.

Reply:
there is no dependency here